src/cmds/qstat.c

```diff
@@ -1070,17 +1191,19 @@ display_statjob(struct batch_status *status, struct batch_status *prtheader, int
 							 * Use a stack variable instead.
 							 */
 							char noval[] = "UNKNOWN";
-							prt_attr(a->name, a->resource, noval);
-							printf("\n");
+							prt_attr(a->name, a->resource, noval, alt_opt & ALT_DISPLAY_1l);
 						} else
-							prt_attr(a->name, a->resource, ctime(&epoch));
+						{
+							char time_buffer[32];
+							strcpy(time_buffer,ctime(&epoch));
+							time_buffer[strlen(time_buffer)-1]='\0';
```

Comment:
ctime() adds a new line at the end. I'm getting rid of that to print the new line only once inside the loop.